<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1345</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>sine</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>square</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>triangle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>rising sawtooth</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>falling sawtooth</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DC voltage</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="frecBox">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>10</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1000</number>
    </property>
    <property name="maximum">
     <number>20000000</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>87</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Block</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>391</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>151</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Input &amp; output signals</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdK">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>21</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>K =</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="WIDTH">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>20</y>
      <width>45</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>605</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>PULL WIDTH</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>18</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="deltaPlot" native="true">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>80</y>
      <width>361</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>20</y>
      <width>87</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>START FFT</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="FTI" native="true">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>340</y>
      <width>391</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="HFunc" native="true">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>340</y>
      <width>361</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>40</y>
      <width>16</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>R=</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="resister">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>40</y>
      <width>45</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>31</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>C=</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="capacity">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>45</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QCustomPlot" name="koeff" native="true">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>80</y>
      <width>351</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>20</y>
      <width>87</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>K</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>66</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>nF</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>66</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>kOhm</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ef">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>10</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="st">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>30</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>10</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>END FREQ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>1085</x>
      <y>30</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>STEP FREQ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>151</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Ranges of oscilloscope</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>20mV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>50mV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100mV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>200mV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>500mV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1V</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2V</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5V</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10V</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20V</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1345</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
